nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/themes/nifi.scss

```diff
@@ -49,355 +53,100 @@ $material-primary-light-palette: (
-$nifi-canvas-accent-light-palette: (
-    // 50 -> 900 are the PRIMARY colors defined by https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors for primary color #52bf7e
-    50: #e6f6ec,
-    100: #c3e8d0, // "lighter" hue for this palette.
-    200: #9dd9b2, //.running
-    300: #73ca94, //.backpressure-percent
-    400: #52bf7e, // color="primary" Default hue for this palette. Also .up-to-date
-    500: #2cb367,
-    600: #1A9964, //.version-control
-    700: #016131, // "darker" hue for this palette Also .bulletin-normal
-    800: #0000ff, //.endpoint, g.process-group.drop rect.border
-    900: #00ff00, //.connectable-destination, .connector.connectable
-
-    // A100 - A700 are the ANALOGOUS colors but are more customized. These colors are used to highlight, warn, denote midpoints and labelpoints, etc
-    A100: #ffef85, //.selected
-    A200: #f8bf47, //.invalid, .is-missing-port, circle.context
-    A400: #bda500, //.backpressure-percent-warning, .bulletin-warn, .backpressure-percent.warning, text.run-status-icon
-    A700: #ffcc00, //g.connection.selected rect.border, .connection-selection-path, .midpoint, .labelpoint
-
-    // These are the $nifi-canvas-accent-light-palette PRIMARY palette contrast colors. These color do not really get defined by https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors.
-    // Instead if we look to the Angular Material provided palettes we see that these fields are typically rgba(black, 0.87) or white. These values are particularly important
-    // for light mode and dark mode as these values set the colors for the text when displayed against the primary background on a button, badge, chip, etc.
-    //
-    // NOTE: Care should be taken here to ensure the values meet accessibility standards.
-    //
-    // NOTE: When creating the Material palette definition mat.define-palette($nifi-canvas-accent-light-palette, 400, 100, 700);
-    // Since 400 is the default the contrast-400 will be used as the default text color in some cases.
-    contrast: (
-        50: rgba(black, 0.87),
-        100: rgba(black, 0.87),
-        200: rgba(black, 0.87),
-        300: rgba(black, 0.87),
-        400: rgba(black, 0.87),
-        500: rgba(black, 0.87),
-        600: #ffffff,
-        700: #ffffff,
-        800: #ffffff,
-        900: #ffffff,
-        A100: rgba(black, 0.87),
-        A200: rgba(black, 0.87),
-        A400: rgba(black, 0.87),
-        A700: rgba(black, 0.87),
+        50: $on-surface-dark,
+        100: $on-surface-dark,
+        200: $on-surface-dark,
+        300: $on-surface-dark, 
+        400: $on-surface-dark,
+        500: $on-surface-dark,
+        600: #004849, // We use a custom contrast color for the primary default color.
+        700: $on-surface-light,
+        800: $on-surface-light,
+        900: $on-surface-light,
+        A100: $on-surface-dark,
+        A200: $on-surface-dark,
+        A400: $on-surface-dark,
```

Comment:
```suggestion
        A400: $on-surface-light,
```

Reply:
Not enough contrast https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=44A3CF